aiohttp/connector.py

```diff
@@ -841,6 +844,7 @@ def __init__(
         timeout_ceil_threshold: float = 5,
         happy_eyeballs_delay: Optional[float] = 0.25,
         interleave: Optional[int] = None,
+        tcp_sockopts: List[_SOCKOPT] = [],
```

Comment:
Don't use mutable defaults. We only need something iterable here:
```suggestion
        tcp_sockopts: Iterable[_SOCKOPT] = (),
```

Reply:
suggestion taken. resolving.

Reply:
It looks like the final commit that has been merged still uses list as default argument.

Reply:
Really? I just updated and see `Iterable` in `7379a866bb`

Reply:
That's the annotation, the default is still a list `[]`. I missed that that in the review.

Reply:
Ah, I see what you're saying now. I can prepare another PR if you'd like, unless somebody is actively working on https://github.com/aio-libs/aiohttp/issues/10520.

What do you prefer as a default? `()` is explicitly read-only but I could see an argument for `None`... I prefer an empty tuple

Reply:
Eh or maybe I'll just give a crack at #10520 if I'm gonna open it up again anyways